cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -36,9 +35,6 @@ import           Cardano.BM.Backend.Switchboard (Switchboard)
 import qualified Cardano.BM.Backend.Switchboard as Switchboard
 import           Cardano.BM.Backend.TraceForwarder (plugin)
 import           Cardano.BM.Configuration (Configuration)
-#ifdef UNIX
```

Comment:
If we're no longer using the `UNIX` macro in this module, let's remove its definition and also the `CPP` language extension.

Reply:
Done. But `CPP` is still required.

Reply:
Right :+1: